### ISSUE TITLE
Bump dependent-sum bound

### ISSUE THREAD

**Issue by abbradar** [mokus0/dependent-sum-template#7]
It seems to work with dependent-sum 0.4.

**Comment by int-e** [comment]
This affects lambdabot, cf. https://github.com/lambdabot/lambdabot/issues/158

How do you feel about this, should we try to get rid of the dependency on dependent-sum-template?

**Comment by abbradar** [comment]
Not sure what should be done because I usually feel bad reimplementing/copying/circumventing something that already exists. Ideally of course the maintainer would show up...

**Comment by afcady** [comment]
To work around this, I put this in my `stack.yaml`:

```
packages:
- '.'
- location:
    git: https://github.com/afcady/dependent-sum-template.git
    commit: 68fc1a90580ea8f20775491cf651eefea422212a
  extra-dep: true
```

**Comment by afcady** [comment]
By the way, 15 days to show up is not really that long of a time.

**Comment by JonCoens** [comment]
It's been a handful of months with no reply. Is dependent-sum-template abandoned?

**Comment by ryantrinkle** [comment]
Resolved by #8.  Thanks for reporting!
